translations/en-us.yml

```
@@ -579,7 +579,7 @@ node_card:
         private_tooltip: 'This registration is private'
         timestamp_label: 'Registered:'
         statuses:
-            revision_pending_moderation: 'Pending moderation'
+            revision_pending_moderation: 'Revision pending moderation'
```

I didn't see this as a language change being requested, but I think specifying that it's an update needing to be updated would be good. With that said, I think we are using `Update` as the user-facing language, so
```suggestion
            revision_pending_moderation: 'Update pending moderation'
```

I will ask Mark if he thinks this would be a good change as well.